app-admin/longrun/longrun-0.9-r4.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
```

Comment:
groff should be a BDEPEND here, drop blank IUSE, maybe check for a newer Debian patch